src/main/java/oss/fosslight/api/controller/v1/ApiProjectController.java

```diff
@@ -1387,4 +1391,19 @@ public CommonResult ossReportAndroid(
 		
 		return responseService.getSingleResult(resultMap);
 	}
+
+	@ApiOperation(value = "Download Project Notice", notes = "Project > Download")
+	@ApiImplicitParams({
+			@ApiImplicitParam(name = "_token", value = "token", required = true, dataType = "String", paramType = "header")
+	})
+	@GetMapping(value = API.FOSSLIGHT_API_PROJECT_NOTICE_DOWNLOAD)
+	public ResponseEntity<FileSystemResource> downloadNotice (
+			@RequestHeader String _token,
+			@ApiParam(value = "Project id", required = true) @RequestParam(required = true) String prjId
+	) throws IOException {
+		userService.checkApiUserAuth(_token);
+		ResponseEntity<FileSystemResource> result = null;
+		result = verificationService.getNotice(prjId, CommonFunction.emptyCheckProperty("notice.path", "/notice"));
+		return result;
```

Comment:
I think that this code is more cleaner

```
        ResponseEntity<FileSystemResource> result =
                verificationService.getNotice(prjId, CommonFunction.emptyCheckProperty("notice.path", "/notice"));

        return result;
```

Reply:
I did this since the original code in `VerificationController` used this style
<img width="1166" alt="image" src="https://github.com/fosslight/fosslight/assets/77962265/f752957a-1e88-485f-85f8-12c8bedf8f2b">

Would it be better to follow that style?
or what about the style like this? (like the other APIs)
```
return verificationService.getNotice(prjId, CommonFunction.emptyCheckProperty("notice.path", "/notice"));
```

Reply:
If other APIs return like that code, it might be a good idea to modify it by adding the service code to the return.